contracts/dmarket.js

```diff
@@ -0,0 +1,1285 @@
+/* eslint-disable no-await-in-loop */
+/* global actions, api */
+
+const UTILITY_TOKEN_SYMBOL = 'BEE';
+
+// 30 days in seconds
+const MAX_EXPIRATION_SECS = 2592000;
+// fee in UTILITY TOKEN required to create a new pair
+const PAIR_CREATION_FEE = '500';
+
+const CONTRACT_NAME = 'dmarket';
+
+actions.createSSC = async () => {
+  const tableExists = await api.db.tableExists('pairs');
+
+  if (tableExists === false) {
+    await api.db.createTable('pairs', ['symbol', 'precision', 'allowedSymbols']);
+    await api.db.createTable('buyBook', ['symbol', 'pair', 'account', 'priceDec', 'expiration', 'txId']);
+    await api.db.createTable('sellBook', ['symbol', 'pair', 'account', 'priceDec', 'expiration', 'txId']);
+    await api.db.createTable('tradesHistory', ['symbol', 'pair']);
+    await api.db.createTable('metrics', ['symbol', 'pair']);
+
+    // default global pairs
+    const pairs = [
```

Comment:
Should these be called "baseTokens"? Pair generally refers to two tokens, e.g. https://coinmarketcap.com/alexandria/glossary/pair

Reply:
Yes, this does make more sense. Not good with naming things.